erpnext_documentation/www/docs/user/manual/en/regional/france/index.md

```diff
@@ -0,0 +1,8 @@
+<!-- add-breadcrumbs -->
+# Requirements for France
+
+ERPNext aims to support local regulation for different regions in the world. In most cases ERPNext is very flexible so you can easily add Custom Fields and make Custom Reports to support the regulation of your region.
```

Comment:
In most cases, ERPNext is very flexible, so you can easily add Custom Fields and make Custom Reports support the regulation of your region.